doc/source/contributing_code.rst

```diff
@@ -761,6 +763,18 @@ A typical function docstring looks like the following:
             """
             return True
 
+.. note::
```

Comment:
It's 2025 and MDAnalysis supports Python 3.10 - 3.13: Could we remove the note and go back to the original doc update?